lib/executor/ramping_vus.go

```diff
@@ -86,6 +86,10 @@ func (vlvc RampingVUsConfig) Validate() []error {
 		errors = append(errors, fmt.Errorf("either startVUs or one of the stages' target values must be greater than 0"))
 	}
 
+	if sumTotalNumberOfUpAndDonwShifts(vlvc.StartVUs.Int64, vlvc.Stages) > 100000000 {
```

Comment:
Without any external context (when we look back at it 6 months from now), this value would feel somewhat arbitrary. The fact that the value is arbitrary is acceptable, but I would prefer if it were stored in a constant with an explicit name instead so that the code states what it does, and if I need to know what the value is or change it, I can consult a single place for that.

I would suggest some constant name along the lines of `MaxRampingVUShift` or something along those lines.